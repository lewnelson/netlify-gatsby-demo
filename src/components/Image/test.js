import React from 'react'
import Image from './'
import { shallow } from 'enzyme'

function getWrapper (propOverrides) {
  return shallow(
    <Image
      {...propOverrides}
    />
  )
}

describe('<Image />', () => {
  test('it renders correctly', () => {
    expect(getWrapper()).toMatchSnapshot()
  })
})
