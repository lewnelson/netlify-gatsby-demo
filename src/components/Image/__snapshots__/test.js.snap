// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> it renders correctly 1`] = `ShallowWrapper {}`;
